plugin/kubernetes/kubernetes.go

```diff
@@ -461,12 +468,3 @@ func match(a, b string) bool {
 func wildcard(s string) bool {
 	return s == "*" || s == "any"
 }
-
-const (
```

Comment:
Please try (I know it is hard) to now do cleanups in the same PR

Reply:
Should I just defer these for a later PR then?

Reply:
yep